airbyte-integrations/connectors/source-freshdesk/source_freshdesk/manifest.yaml

```diff
@@ -0,0 +1,488 @@
+version: 0.57.0
+type: DeclarativeSource
+check:
+  type: CheckStream
+  stream_names:
+    - agents
+
+spec:
+  type: Spec
+  connection_specification:
+    type: object
+    $schema: http://json-schema.org/draft-07/schema#
+    required:
+      - api_key
+      - domain
+    properties:
+      api_key:
+        type: string
+        order: 1
+        title: API Key
+        description: 'Freshdesk API Key. See the <a href="https://docs.airbyte.com/integrations/sources/freshdesk">docs</a> for more information on how to obtain this key.'
+        airbyte_secret: true
+      domain:
+        type: string
+        order: 2
+        title: Domain
+        description: "Freshdesk domain"
+        examples:
+          - "myaccount.freshdesk.com"
+        pattern: "^[a-zA-Z0-9._-]*\\.freshdesk\\.com$"
+      start_date:
+        title: "Start Date"
+        type: "string"
+        order: 3
+        description: "UTC date and time. Any data created after this date will be replicated. If this parameter is not set, all data will be replicated."
+        format: "date-time"
+        pattern: "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$"
+        examples:
+          - "2020-12-01T00:00:00Z"
+      requests_per_minute:
+        type: integer
+        order: 4
+        title: "Requests per minute"
+        description: "The number of requests per minute that this source allowed to use. There is a rate limit of 50 requests per minute per app per account."
+      lookback_window:
```

Comment:
[The Satisfaction Ratings endpoint](https://developers.freshdesk.com/api/#view_all_satisfaction_ratings) only supports filtering by created time, so a lookback window was added to avoid missing data.
Fixed name to `lookback_window_in_days`.

Reply:
How does this avoid missing data? Is it because users know after how many days the records can't be updated anymore?

Reply:
@maxi297 Sorry, I didn't mean no data will be missing; this is implemented according to [the incremental sync cookbook](https://www.notion.so/The-Incremental-Sync-Cookbook-b3fe070dd17e4cac940f2a079fc79f4c). The lookback window will be used to find potential updates to recently created items.

Reply:
Ok! I didn't know it was the logic in our cookbook. A couple of questions here:
* Does that mean that the cursor_field should be `created_at` and not `updated_at` https://github.com/airbytehq/airbyte/pull/35699/files#diff-3215e5a55e452c94e4f10c215fb9185778e1fb361b1e6a52f56ce2e7efe0cfcfR302
* Can we be clearer as to why we have this? Something like "Incremental syncs for satisfaction_ratings rely on the created_at field at the API does not allow to filter using update_at. The `lookback_window_in_days` allows for the user to define a time range in which satisfaction_ratings can be updated to ensure updates are also capture in the sync. Given that you don't know this date, we recommend using full refresh to prevent data loss.

Reply:
@maxi297 
1) Yes, the correct name would be `created_at`, I didn't change the name to avoid breaking changes. It would be great to migrate the state as it is an easy change.
2) I updated [the docs](https://github.com/airbytehq/airbyte/pull/35699/files#diff-0630547b1fb8d6ae1759d15837e7c78ff4a4b5c28e7a5f54cb1b25339cf6b48aR19):
```
9. For **Lookback Window**, you may specify a number of days back from the current stream state to re-read data for the **Satisfaction Ratings** stream. This helps capture updates made to existing ratings after their initial creation. However, keep in mind that records updated before (stream_state - lookback_window) won't be synced. To ensure no data loss, we recommend using a full refresh. The default lookback window is set to 14 days.
```

Reply:
> I didn't change the name to avoid breaking changes. It would be great to migrate the state as it is an easy change

I'm fine with creating an issue and talking this outside of the low-code migration to keep the ability to revert the low-code migration if necessary. Can you create the issue and paste it here for documentation purposes?